ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -1136,7 +1136,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi
-- as: gcp-ipi-longduration-reliability-test-f60
+- as: gcp-ipi-f60-sanity-reliability-test
```

Comment:
Please keep a tag `longrun` for these jobs, we can get the job attribute from name.